docs/settings/linux/tasks.json

```diff
@@ -183,24 +178,14 @@
     },
     {
       "type": "shell",
-      "label": "ibus-keyman: build tests",
-      "command": "make",
-      "options": {
-        "cwd": "${workspaceFolder}/linux/ibus-keyman/tests",
-      },
-      "group": "build",
-      "detail": "build ibus-keyman"
-    },
-    {
-      "type": "shell",
-      "label": "ibus-keyman: check",
-      "command": "make",
-      "args": ["check"],
+      "label": "ibus-keyman: test",
+      "command": "./build.sh",
+      "args": ["test", "--debug"],
       "options": {
         "cwd": "${workspaceFolder}/linux/ibus-keyman/",
       },
       "group": "build",
-      "detail": "build check ibus-keyman"
+      "detail": "run tests of ibus-keyman"
```

Comment:
Do you want to install an `install` task?

Reply:
No, I don't see a need for that.